src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -765,30 +802,65 @@ void CommandRequestImpl::AddDissalowedParameterToInfoString(
 
 void CommandRequestImpl::AddDisallowedParametersToInfo(
     smart_objects::SmartObject& response) const {
-  std::string info;
+  const mobile_apis::FunctionID::eType id =
+      static_cast<mobile_apis::FunctionID::eType>(function_id());
+
+  if (!helpers::Compare<mobile_apis::FunctionID::eType,
+                        helpers::EQ,
+                        helpers::ONE>(
+          id,
+          mobile_apis::FunctionID::SubscribeVehicleDataID,
+          mobile_apis::FunctionID::UnsubscribeVehicleDataID,
+          mobile_apis::FunctionID::GetVehicleDataID,
+          mobile_apis::FunctionID::SendLocationID)) {
+    return;
+  }
+
+  std::string disallowed_by_user_info;
+  InfoAdder user_info_adder(disallowed_by_user_info);
+
+  std::for_each(removed_parameters_permissions_.disallowed_params.begin(),
+                removed_parameters_permissions_.disallowed_params.end(),
+                user_info_adder);
+
+  if (!disallowed_by_user_info.empty()) {
+    disallowed_by_user_info +=
+        1 < removed_parameters_permissions_.disallowed_params.size() ? " are"
+                                                                     : " is";
+    disallowed_by_user_info += " disallowed by user";
+  }
 
-  RPCParams::const_iterator it =
-      removed_parameters_permissions_.disallowed_params.begin();
-  for (; it != removed_parameters_permissions_.disallowed_params.end(); ++it) {
-    AddDissalowedParameterToInfoString(info, (*it));
+  std::string disallowed_by_policy_info;
+  InfoAdder policy_info_adder(disallowed_by_policy_info);
+
+  std::for_each(removed_parameters_permissions_.undefined_params.begin(),
+                removed_parameters_permissions_.undefined_params.end(),
+                policy_info_adder);
+
+  if (!disallowed_by_policy_info.empty()) {
+    disallowed_by_policy_info +=
+        1 < removed_parameters_permissions_.undefined_params.size() ? " are"
```

Comment:
the same

Reply:
@KVGrygoriev fixed at ed771ed2423da63dd4a93695aa925edc47cf74f9